_posts/2018-03-10-meetup-march.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Meetup Summary (March, 2018)
+---
+
+I reckon this is the first post in a long while on this website.
+We've been having meetups for the last one year, but most of the
+things discussed are lost in time because there is no record of
+it. I thus felt I needed to put some of the stuff discussed on our
```

Comment:
_some of the stuff discussed ~~on~~ **in** our ..._

Reply:
Fix that typo. For now, I feel the article is good 'nuff; I'll merge changes later.

Reply:
Hi. I just fixed the typo